src/components/large-countdown-clock/LargeCountdownClock.js

```diff
@@ -0,0 +1,254 @@
+      {completed ? (
+        <CheckmarkAnimation />
+      ) : (
+        <CountdownContainer exiting={clockExiting}>
+          {displayMinutes ? (
+            <Clock
+              exiting={minutesExiting}
+              key="withMinutes"
+              uglyDigitOffset={displayMinutes === 1}
+            >
+              <ClockMinutes
+                allowFontScaling={false}
+                lineHeight={lineHeight}
+                size={size}
+              >
+                {displayMinutes}
+              </ClockMinutes>
+              <SeparatorDots
+                minuteEndsWithOne={minuteEndsWithOne}
+                size={separatorSize}
+              />
+              <ClockSeconds
+                allowFontScaling={false}
+                lineHeight={lineHeight}
+                size={size}
+              >
+                {displaySeconds}
+              </ClockSeconds>
+            </Clock>
+          ) : (
+            <Clock
+              entering={entering}
+              key="onlySeconds"
+              uglyDigitOffset={displaySeconds >= 10 && displaySeconds < 20}
+            >
+              <ClockSeconds allowFontScaling={false} lineHeight={57} size={50}>
+                {displaySeconds}
+              </ClockSeconds>
+            </Clock>
+          )}
+          <Svg height="132" viewBox="0 0 130 130" width="132">
+            <Defs>
+              <RadialGradient
+                cx="50%"
+                cy="50%"
+                fx="50%"
+                fy="50%"
+                id="large-countdown-clock-fill"
+                r="50%"
+              >
+                <Stop offset="0%" stopColor="#9875D7" stopOpacity="0" />
+                <Stop offset="100%" stopColor="#9875D7" stopOpacity="0.06" />
+              </RadialGradient>
+            </Defs>
+            <G>
+              <Circle
+                cx={PROGRESS_CENTER_COORDINATE}
+                cy={PROGRESS_CENTER_COORDINATE}
+                fill="url(#large-countdown-clock-fill)"
+                id="large-countdown-clock-center-face"
+                r="51"
+              />
+              <AnimatedCircle
+                animatedProps={animatedStroke}
+                cx={PROGRESS_CENTER_COORDINATE}
+                cy={PROGRESS_CENTER_COORDINATE}
+                id="large-countdown-clock-progress-bar"
+                r={PROGRESS_RADIUS}
+                stroke="#9875D7"
+                strokeDasharray={PROGRESS_STROKE_FULL_LENGTH}
+                strokeDashoffset={0}
+                strokeLinecap="round"
+                strokeWidth={PROGRESS_STROKE_WIDTH}
+                transform={`rotate(-90 ${PROGRESS_CENTER_COORDINATE} ${PROGRESS_CENTER_COORDINATE})`}
+              />
+            </G>
+          </Svg>
+        </CountdownContainer>
+      )}
+    </Scene>
+  );
+}
+
+const Scene = styled(Animated.createAnimatedComponent(Flex))({
+  alignItems: 'center',
+  height: 132,
+  justifyContent: 'center',
+  marginBottom: 40,
```

Comment:
Avoid using margin within components pls 🙏 this should be handled by the parent layout within `ENSConfirmRegisterSheet` and not the component itself.

Reply:
Ahh leftover from sandboxing. My bad